lib/theme/app_colors.dart

```diff
@@ -4,4 +4,5 @@ class AppColors {
   static const blackRussian = Color(0xFF15151A);
   static const whiteAlpha18 = Color(0x2EFFFFFF);
   static const whiteAlpha50 = Color(0x80FFFFFF);
+  static const nero90 = Color(0xE61E1E1E);
```

Comment:
@AVI5HEK May I know how we're different between these prefixes: `0xFF` and `0xE6`? How do you know and pick them🙏 

For `whiteAlpha18` and `whiteAlpha50` We can apply from `Colors.white` for example: Colors.white.withAlpha(50)

Reply:
@Wadeewee [here's](https://davidwalsh.name/hex-opacity) a cheat sheet to find the hex value for the opacity.

Reply:
@AVI5HEK If we add them in the `XML` and let them auto-generate we don't worry about hex code anymore 🙏 https://github.com/nimblehq/ic-flutter-avishek/pull/40#discussion_r1270247014

Reply:
@Wadeewee but we are using hex code in the XML file too. 🤔 For example, the value of this color would be `#E61E1E1E`.

Reply:
Noted: You prefer to go with way and I'm ok 🚀 For more info we ask in `flutter_template` later